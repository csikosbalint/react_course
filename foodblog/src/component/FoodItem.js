import React from 'react';
import { ListGroup } from 'react-bootstrap';

export default function FoodItem(props) {
    let href = `/food/${props.food.id}`
    return (
        <ListGroup.Item action href={href}>{props.food.name}</ListGroup.Item>
    )
}