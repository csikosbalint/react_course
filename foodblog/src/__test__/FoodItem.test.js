import React from 'react';
import {create} from 'react-test-renderer';
import FoodItem from '../component/FoodItem';

describe('FoodItem Component Snapshot', () => {
    test("FoodItem",() => {
        let tree = create(<FoodItem food={{ name: "alma"}}/>);
        expect(tree.toJSON()).toMatchSnapshot();
    })
})
