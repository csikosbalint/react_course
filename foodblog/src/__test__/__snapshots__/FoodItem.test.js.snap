// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FoodItem Component Snapshot FoodItem 1`] = `
<a
  className="list-group-item list-group-item-action"
  disabled={false}
  href="/food/undefined"
  onClick={[Function]}
>
  alma
</a>
`;
